tests/specs/graph/JsrSpecifiers_FastCheckCache_DiagnosticThenNestedDep.txt

```diff
@@ -0,0 +1,191 @@
+~~ {"fastCheckCache":true} ~~
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{ "exports": { ".": "./mod.ts" } }
+
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+export function add(a: number, b: number) {
+  return Math.random();
+}
+
+export * from "./a.ts";
+
+# https://jsr.io/@scope/a/1.0.0/a.ts
+// should still grab this as a dep for this module for fast check
+export * from "jsr:@scope/b@1";
+
+// even though this is private, it should still grab this as a
+// dependency because we couldn't fast check this current package
+// todo(#391): fix this
+import { Private } from "jsr:@scope/c@1";
+console.log(Private);
```

Comment:
Since fast check failed on this package, it should still grab this dependency and `jsr:@scope/c@1.0.0` should be fast checked. Fixing this is a perf improvement and I'll do it in a future PR. I opened https://github.com/denoland/deno_graph/issues/391